frontend/src/features/customer/components/cards/HelperSelectInfoCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { IcAlertCircle } from '@icons';
+
+const HelperSelectInfoCard = () => {
+  return (
+    <div className='bg-neutral-10 flex flex-col gap-[1rem] rounded-[0.8rem] p-[1.2rem]'>
+      <div className='flex-start gap-[0.8rem]'>
```

Comment:
피그마 상으로는 0.4rem인 것 같습니다!
```suggestion
      <div className='flex-start gap-[0.4rem]'>
```

Reply:
수정했습니다!